app/src/ui/tutorial/tutorial-panel.tsx

```diff
@@ -115,17 +116,17 @@ export class TutorialPanel extends React.Component<
                   It doesn’t look like you have a text editor installed. We can
                   recommend{' '}
                   <LinkButton
-                    uri="https://atom.io"
-                    title="Open the Atom website"
+                    uri={suggestedExternalEditor.url}
+                    title={`Open the ${suggestedExternalEditor.name} Code website`}
```

Comment:
There is a leftover here, right?
```suggestion
                    title={`Open the ${suggestedExternalEditor.name} website`}
```